tools/topology/platform/intel/intel-generic-dmic-kwd.m4

```diff
@@ -0,0 +1,121 @@
+#
+# Topology for digital microphones array
+#
+
+include(`platform/intel/dmic.m4')
+
+# variable that need to be defined in upper m4
+# define(CHANNELS, `4')
+
+# define(DMIC_PCM_48k_ID, `99')
+# define(DMIC_PIPELINE_48k_ID, `4')
+# define(DMIC_DAI_LINK_48k_ID, `1')
+
+# define(DMIC_PCM_16k_ID, `7')
```

Comment:
Hi @xiulipan ,

Can we make the PCM ID for DMIC in an incremental order? like (DMIC_PCM_48k_ID, `99') and (DMIC_PCM_16k_ID, `100').

Reply:
OK, will send an update.